lib/SnakesLadders.rb

```diff
@@ -2,16 +2,15 @@
 require File.dirname(__FILE__) + '/Space'
 
 class SnakesLadders
-    # declare instance variables
-    @players
-    @spaces
-    @is_finished
```

Comment:
I got rid of the declarations here. Initializing declares the vars automatically